com.unity.render-pipelines.universal/ShaderLibrary/Debug/DebugViewEnums.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+
+namespace UnityEngine.Rendering.Universal
+{
+    #region Material Settings
+    [GenerateHLSL]
+    public enum DebugMaterialMode
+    {
+        None,
+        Albedo,
+        Specular,
+        Alpha,
+        Smoothness,
+        AmbientOcclusion,
+        Emission,
+        NormalWorldSpace,
+        NormalTangentSpace,
+        LightingComplexity,
+        LOD,
+        Metallic,
+        SpriteMask,
+    }
+
+    [GenerateHLSL]
+    public enum DebugVertexAttributeMode
+    {
+        None,
+        Texcoord0,
+        Texcoord1,
+        Texcoord2,
+        Texcoord3,
+        Color,
+        Tangent,
+        Normal,
+    }
+    #endregion
+
+    #region Rendering Settings
+    [GenerateHLSL]
+    public enum DebugFullScreenMode
+    {
+        None,
+        Depth,
+
+        // TODO: Restore this once we have access to the screen-space shadow texture...
+        //MainLightShadowsOnly,
```

Comment:
once you restore this then the numbering will be different, It's either you add to the end or maybe it's already ok to enable this one.

Reply:
@andrewt-unity do you know if this is possible to implement? Or should we just remove the commented code? Not sure what this is waiting for.

For the numbering issue, I wouldn't have thought it's an issue since these debug enums are never serialized anywhere.

Reply:
@phi-lira would know but I think we could just remove it for now - we just need to remember to reimplement at some point, but that's assuming we can get access to the necessary data (which we couldn't at the point we removed it)

Reply:
I've removed them to keep checked in code clean.